cosmwasm/contracts/global-accountant/src/msg.rs

```diff
@@ -139,7 +116,17 @@ pub enum ExecuteMsg {
 }
 
 #[cw_serde]
-pub struct MigrateMsg {}
+pub struct MigrateMsg {
+    // Optional list modifications to apply
+    pub modifications: Vec<ModifyBalance>,
+}
+
+/// Modifies the balance of a single account.  Used to manually override the balance.
+#[cw_serde]
+pub struct ModifyBalance {
+    // A serialized `Modification` message.
+    pub modification: Binary,
```

Comment:
This was a serialized message because we needed to verify signatures but since that's taken care of we should be able to use `accountant::Modification` directly in the message body.